adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/X509AuthHandler.java

```diff
@@ -190,8 +190,7 @@ public final void parseCredentials(final RoutingContext context, final Handler<A
                 });
             } catch (SSLPeerUnverifiedException e) {
                 // client certificate has not been validated
-                TracingHelper.logError(span, e);
-                span.finish();
+                LOG.debug("could not get client certificates: " + e.getMessage());
```

Comment:
can you use the usual placeholder syntax for log statements, i.e.
`LOG.debug("could not get client certificates: {}", e.getMessage())`

Reply:
yes, I've changed that.